src/main/java/thePackmaster/patches/clawpack/ClawUsePatch.java

```diff
@@ -10,16 +12,20 @@
 import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
 import com.megacrit.cardcrawl.monsters.AbstractMonster;
 import thePackmaster.SpireAnniversary5Mod;
-import thePackmaster.powers.clawpack.ClawStrengthPower;
+import thePackmaster.cards.clawpack.AbstractClawCard;
 import thePackmaster.util.Wiz;
 
+import java.util.Iterator;
+
 @SpirePatch(
         clz = GashAction.class,
         method = "update"
 )
 public class ClawUsePatch {
-        public static SpireReturn Postfix(GashAction __instance) {
-            Wiz.applyToSelf(new ClawStrengthPower(Wiz.p(), __instance.amount));
-            return SpireReturn.Return();
-        }
+
+    @SpirePrefixPatch
+    public static void Prefix(GashAction __instance) {
+        AbstractClawCard.ClawUp(__instance.amount, true);
```

Comment:
Yeah, this seems like a fine way to do it. The version that doesn't touch base game Claws is called here (since GashAction already handles that), while the version that does touch base game Claws is called everywhere else.

Glad we made this change.